pkg/storage/gcs.go

```diff
@@ -139,6 +139,14 @@ func (s *gcsStorage) FileExists(ctx context.Context, name string) (bool, error)
 }
 
 func newGCSStorage(ctx context.Context, gcs *backup.GCS) (*gcsStorage, error) {
+	if gcs.CredentialsBlob == "" {
```

Comment:
```suggestion
	if gcs.CredentialsBlob == ""  && sendCredential {
```

Reply:
No, it's not related to `sendCredential`.
We should get credentials from environment if user does not provide credentials explicitly.